dbt/include/duckdb/macros/utils/external_location.sql

```diff
@@ -1,3 +1,3 @@
-{%- macro external_location(format) -%}
+{%- macro external_location(this, format) -%}
```

Comment:
Does this arg need to be `this` vs., say, `relation`? Like is there any downside to renaming it, aside from the extra line in the file that needs to change?

Reply:
I originally had `relation` instead of `this`, but reasoned that if someone had a external_location macro and only changed the signature to match instead of changing `this` to `relation` within the macro definition, then very confusing things might result (i.e. we would be registering in the current node as the upstream node). Hence the change to shadow `this` to avoid that possibility.

If this reasoning seems baroque, happy to change back to `relation`.

Reply:
It seems baroque, yeah-- I'm not worried about someone who has the sophistication to override the macro not having the sophistication to update the reference

Reply:
sounds good, I've reset the branch to the previous commit, removing this shadowing